recipes/gtest/all/conanfile.py

```diff
@@ -145,8 +145,8 @@ def _postfix(self):
     def package_info(self):
 
         self.cpp_info.set_property("cmake_file_name", "GTest")
-        self.cpp_info.set_property("cmake_target_name", "GTest")
-        self.cpp_info.components["libgtest"].set_property("cmake_target_name", "gtest")
+        self.cpp_info.set_property("cmake_target_name", "GTest::GTest")
```

Comment:
Actually `GTest::GTest` is an alias of `GTest::gtest`, so it shouldn't depend on all components.
As well, `GTest::Main` is deprecated, it could be added as an alias of `GTest::gtest_main`